cloudify_ansible/utils.py

```diff
@@ -19,6 +19,7 @@
 import yaml
 import errno
 import shutil
+import subprocess
```

Comment:
Let keep the consistency of using the local runner , as other parts are already using it